### ISSUE TITLE
Modify instruction of challenge Comparison with the Less Than Or Equal To Operator

### ISSUE THREAD

**Issue by 1810msr** [FreeCodeCamp/FreeCodeCamp#11663]
Challenge [Comparison with the Less Than Or Equal To Operator](https://www.freecodecamp.com/challenges/comparison-with-the-less-than-or-equal-to-operator#?solution=%0Afunction%20testLessOrEqual(val)%20%7B%0A%20%20if%20(val%20%3C%3D%2012)%20%7B%20%20%2F%2F%20Change%20this%20line%0A%20%20%20%20return%20%22Smaller%20Than%20or%20Equal%20to%2012%22%3B%0A%20%20%7D%0A%20%20%0A%20%20if%20(val%20%3C%3D%2024)%20%7B%20%20%2F%2F%20Change%20this%20line%0A%20%20%20%20return%20%22Smaller%20Than%20or%20Equal%20to%2024%22%3B%0A%20%20%7D%0A%0A%20%20return%20%2225%20or%20More%22%3B%0A%7D%0A%0A%2F%2F%20Change%20this%20value%20to%20test%0AtestLessOrEqual(24.99)%3B%0A%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.125 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript

function testLessOrEqual(val) {
  if (val <= 12) {  // Change this line
    return "Smaller Than or Equal to 12";
  }
  
  if (val <= 24) {  // Change this line
    return "Smaller Than or Equal to 24";
  }

  return "25 or More";
}

// Change this value to test
testLessOrEqual(24.99);


```
![image](https://cloud.githubusercontent.com/assets/23392198/20288353/8b52fea8-ab1e-11e6-8604-d6dddeb3fa54.png)

Issue:
when I input testLessOrEqual(24.99); the answer or the code returns "25 or more " as the outcome which is not correct - this code might need some improvement

**Comment by Pasayadaan** [comment]
@1810msr  The code is behaving correctly as coded though the intention may be different! 
Since you are passing 24.99, the next line shown below will not get satisfied (i.e. 24.99 is not less than 24) so the code will jump to the next line which is "25 or More"
if (val <= 24) {  // Change this line
    return "Smaller Than or Equal to 24";
  }
So change the above condition to 25 and accordingly change the last return.

**Comment by kevcomedia** [comment]
I think the issue is with the string `"25 or More"`. If `val <= 24` fails in the if-condition, then `val` must be greater than 24. Perhaps `"More than 24"` is a more appropriate return value.

**Comment by Bouncey** [comment]
Please can we have the challenge instructions adjusted to `return "More than 24"` if `val` is greater than the integer 24.

**Comment by robbawebba** [comment]
I would love to give this a shot!

**Comment by sukhesh035** [comment]
Write the final return statement as  return "24.1 or More";. Your condition is just checking below 24. But your value which is 24.99 doesn't satisfy any of the above condition and just returning the String you wrote. So write **return "24.1 or more"** which makes more sense

**Comment by robbawebba** [comment]
I believe "More Than 24" is a more accurate return value because the values
that make it to that else statement can be as close as possible to 24
without equaling 24 when approaching from the positive side. If I were to
change the return value to "24.1 or more" and I passed the function a value
of 24.05, 24.01, 24.001, the function would return the statement saying
that the number is larger than 24.1, which is false. Also one of the
moderators suggested saying "More Than 24" in the issue that I referenced
in the description, so I just followed their suggestion.
